app/renderer/controls/Button.tsx

```diff
@@ -19,12 +19,17 @@ const StyledButton = styled.span<StyledButtonProps>`
   background-color: ${(p) => p.theme.buttonPrimaryBg};
   color: ${(p) => p.theme.buttonPrimaryText};
   border-radius: 10px;
+  border: none;
   text-align: center;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   font-size: 0.9em;
 
+  &:focus-visible {
+    outline: ${(p) =>
+      `4px solid ${Color(p.theme.buttonPrimaryBg).alpha(0.5).toString()};`}
```

Comment:
I think there's a missing curly brace here! Also, you could write the outline as:
```css
outline: 4px solid
      ${(p) => Color(p.theme.buttonPrimaryBg).alpha(0.5).toString()};
```

No need for the template literals.

Reply:
Yikes, I'm not sure what I was thinking when I put that template literals there. 😬 

I have corrected that, removed the onClick handler, and cleaned up a few other small details (import order, curly brackets around string literal) that ESLint was complaining about.